4_advanced_usage/77_distributed_rudder.txt

```diff
@@ -73,6 +73,106 @@ cd rudder-multiserver-setup
 
 ----
 
+==== Install rudder-db manually
+
+The setup is done in two places: on the database server and on the Rudder root server.
+It also allows moving an existing database to another server.
+
+===== On the database server
+
+* Install and configure the agent on the node
+
+* Install the *rudder-reports* package
+
+* Change postgresql configuration (postgresql.conf) to listen on the right interface to communicate with the server:
+
+----
+
+listen_addresses = 'IP_TO_USE' 
+----
+
+* Add an authorization line for the server (in pg_hba.conf):
+
+----
+
+host    all         all         IP_OR_NETWORK_INCLUDING_THE_SERVER  trust
+
+----
+
+* Execute the following commnd to configure the password (that should be the same than in `/opt/rudder/etc/rudder-passwords.conf` on the server) :
+
+----
+
+su - postgres -c "psql -c \"ALTER USER rudder WITH PASSWORD 'RUDDER_SERVER_DATABASE_PASSWORD'\""
+
+----
+
+* Restart postgresql to apply the new settings
+
+----
+
+service postgresql restart
+
+----
+
+===== On the root server
+
+* Remove the rudder-server-root and rudder-reports packages if installed. You can also remove the postgresql package and database from the server if existing.
+
+* Change the `/opt/rudder/etc/server-roles.d/rudder-db` file with
+
+----
+
+DATABSE_HOST,SERVER_HOST
+
+----
+
+* Change the hostname in `/opt/rudder/etc/rudder-web.properties`
+
+----
+
+rudder.jdbc.url=jdbc:postgresql://DATABASE_HOST:5432/rudder
+
+----
+
+* Edit `/var/rudder/cfengine-community/inputs/rudder-server-roles.conf` and set the following line:
+
+----
+
+rudder-db:DATABASE_HOST
+
+----
+
+* Edit the /etc/rsyslog.d/rudder.conf file and change the hostname in:
+
+----
+
+:ompgsql:DATABASE_HOST,rudder,rudder,...
```

Comment:
Please clarify that you should replace "DATABASE_HOST" with the IP/hostname of your DB server